fmriprep/config.py

```diff
@@ -589,7 +599,7 @@ class workflow(_Config):
     level = None
     """Level of preprocessing to complete. One of ['minimal', 'resampling', 'full']."""
     longitudinal = False
-    """Run FreeSurfer ``recon-all`` with the ``-logitudinal`` flag."""
+    """*Deprecated* - run FreeSurfer ``recon-all`` with the ``-logitudinal`` flag."""
```

Comment:
Isn't the old description incorrect? I thought the longitudinal Freesurfer workflow wasn't supported.

Reply:
changed, although as i think about it as long as we keep the parser argument and remap to unbiased, this can be removed...